app.js

```diff
@@ -31,6 +31,9 @@ app.use(function(err, req, res, next) {
   res.locals.message = err.message;
   res.locals.error = req.app.get('env') === 'development' ? err : {};
 
+  // print error log to terminal
+  console.log(err.stack)
```

Comment:
If there's no error, you can remove console.log().

Reply:
Hix. Do chưa cấu hình ESLint nên vẫn có thể commit code mà còn `console.log`.

Reply:
Thực ra trong giai đoạn code mình vẫn có thể dùng console.log anh à, có thể để debug. Còn em nghĩ ESLint thì cũng không liên quan lắm.

Reply:
Ko nên dùng console.log đâu Tùng ơi, server mình sẽ sùng logger ra file. Còn eslint như a Trung nói thì nó sẽ tự động báo eslint error khi em dùng console.log. Cái này lần tới a sẽ config luôn cho project của mình.

Reply:
OK anh cái này em cũng chưa biết, em bị lỗi mà chả biết lỗi là gì nên phải log ra :)), thế ra là nó log ra file hả

Reply:
log để debug thì tìm lỗi xong phải xóa console log đi chứ?